src/applications/coronavirus-research/containers/CustomReviewField.jsx

```diff
@@ -33,12 +33,10 @@ export default function CustomReviewField({ children, uiSchema }) {
   }
 
   return children?.props.formData ? (
-    <div>
```

Comment:
If we remove the extra unclassed DIV and just return the `div[class="review-row"]` and the child elements, we can clear out the axe error without having to adjust any other elements.

```html
<!-- CustomReviewField.jsx#36 -->
<div className="review-row">
  <dt>{selectQuestionTitle}:</dt>
  <dd>{uiSchema['ui:title']}</dd>
</div>
```